Tests/js/views/tabs/assets/ez-locationviewdetailstabview-tests.js

```diff
@@ -201,6 +254,145 @@ YUI.add('ez-locationviewdetailstabview-tests', function (Y) {
         },
     });
 
+    orderingTest = new Y.Test.Case({
+        name: "eZ LocationViewDetailsTabView change event test",
+        setUp: function () {
+            var that = this;
+
+            this.sortField = 'PRIORITY';
+            this.sortOrder = 'ASC';
+            this.contentMock = new Mock();
+            this.versionMock = new Mock();
+            this.locationMock = new Mock();
+            this.translationsList = ['eng-GB', 'fre-FR'];
+            this.creator = "13";
+            this.owner = "42";
+
+            Mock.expect(this.locationMock, {
+                method: 'get',
+                args: [Y.Mock.Value.String],
+                run: function (attr) {
+                    if (attr === 'sortField') {
+                        return that.sortField;
+                    } else if (attr === 'sortOrder') {
+                        return that.sortOrder;
+                    } else {
+                        Y.fail("Unexpected parameter " + attr + " for content mock");
+                    }
+                }
+            });
+            
+            Mock.expect(that.contentMock, {
+                method: 'get',
+                args: [Mock.Value.String],
+                run: function (attr) {
+                    if (attr === 'currentVersion') {
+                        return that.versionMock;
+                    } else if (attr === 'resources') {
+                        return {Owner: that.owner};
+                    } else {
+                        Y.fail("Unexpected parameter " + attr + " for content mock");
+                    }
+                }
+            });
+
+            Mock.expect(this.contentMock, {
+                method: 'toJSON',
+                returns: {}
+            });
+
+            Mock.expect(that.versionMock, {
+                method: 'get',
+                args: ['resources'],
+                returns: {Creator: this.creator}
+            });
+
+            Mock.expect(this.versionMock, {
+                method: 'getTranslationsList',
+                returns: this.translationsList
+            });
+
+            Mock.expect(this.versionMock, {
+                method: 'toJSON',
+                returns: {}
+            });
+
+            Mock.expect(this.locationMock, {
+                method: 'toJSON',
+                returns: {}
+            });
+
+
+            this.view = new Y.eZ.LocationViewDetailsTabView({
+                content: this.contentMock,
+                location: this.locationMock,
+                selectedSortField: '',
+                selectedSortOrder: '',
+                container: '.container'
+            });
+        },
+
+        tearDown: function () {
+            this.view.destroy();
+            delete this.view;
+        },
+
+
+        "Should set the selectedSortField and selectedSortOrder attributes": function () {
+            Assert.areSame(
+                this.sortField,
```

Comment:
this is kind of hardcoded. You should actually compare the value of the selected option with the value of the corresponding view attribute.